tests/manage/pv_services/pv_encryption/test_pvc_with_tc_overrie_vn.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+)
+
+from ocs_ci.helpers.helpers import (
+    create_unique_resource_name,
+)
+
+
+log = logging.getLogger(__name__)
+
+kmsprovider = constants.VAULT_KMS_PROVIDER
+argnames = ["kv_version", "kms_provider", "use_vault_namespace"]
+if config.ENV_DATA.get("vault_hcp"):
+    argvalues = [
+        pytest.param(
+            "v1", kmsprovider, True, marks=pytest.mark.polarion_id("OCS-4639")
+        ),
+        pytest.param(
+            "v2", kmsprovider, True, marks=pytest.mark.polarion_id("OCS-4641")
+        ),
+    ]
+else:
+    argvalues = [
+        pytest.param(
+            "v1", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-4638")
+        ),
+        pytest.param(
+            "v2", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-4640")
+        ),
+    ]
+
+
+@pytest.mark.parametrize(
+    argnames=argnames,
+    argvalues=argvalues,
+)
+@tier1
+@skipif_ocs_version("<4.8")
+class TestEncryptedRbdBlockPvcSnapshot(ManageTest):
+    """
+    Tests to check Tenant configmap override vault namespace or not
+    1.
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(
+        self,
+        kv_version,
+        kms_provider,
+        use_vault_namespace,
+        pv_encryption_kms_setup_factory,
+        project_factory,
+        storageclass_factory,
+        pvc_factory_class,
+    ):
+        """
+        Setup csi-kms-connection-details configmap
+
+        """
+
+        log.info("Setting up csi-kms-connection-details configmap")
+        self.kms = pv_encryption_kms_setup_factory(kv_version, use_vault_namespace)
+        log.info("csi-kms-connection-details setup successful")
+
+        # Create a project
+        self.proj_obj = project_factory()
+
+        # Create an encryption enabled storage class for RBD
+        self.sc_obj = storageclass_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            encrypted=True,
+            encryption_kms_id=self.kms.kmsid,
+        )
+
+    def test_status_with_tenant_configmap(self, multi_pvc_factory, kv_version):
```

Comment:
The test name doesn't reflect what the test case is about

Reply:
Corrected name